fgrpc/grpcrunner.go

```diff
@@ -29,6 +29,8 @@ import (
 	"istio.io/fortio/fnet"
 	"istio.io/fortio/log"
 	"istio.io/fortio/periodic"
+	"strings"
+	"regexp"
```

Comment:
let’s not drag in regexp for a prefix match?

Reply:
ack. it felt like a hammer. thanks for calling it out. removed.